### ISSUE TITLE
Collection Report JSON and TSV download links return invalid data

### ISSUE THREAD

**Issue by dietervu** [clarin-eric/curation-dashboard#160]
Both https://curation.clarin.eu/download/collection/AllCollectionReport?format=tsv and https://curation.clarin.eu/download/collection/AllCollectionReport?format=json return invalid files to me. 

The TSV file does not contain a single tab, for the JSON firefox complains: _SyntaxError: JSON.parse: expected double-quoted property name at line 14 column 5 of the JSON data_

**Comment by twagoo** [comment]
JSON does not allow a trailing comma after the last item of an object, but the xml2json stylesheet does appear to generate that. Maybe we can look for an alternative stylesheet?

```
{
        .......... ,{
                "name" : "collection",
    "count" : "0",
    "avgCoverage" : "0.0",
    
    }, ........
```

**Comment by wowasa** [comment]
the TSV can't be correct because of issue #161

**Comment by wowasa** [comment]
> JSON does not allow a trailing comma after the last item of an object, but the [xml2json stylesheet](https://github.com/clarin-eric/curation-dashboard/blob/4bd31d8f80bd4287a23881f5138e9cafa0489b17/curation-web/src/main/resources/xslt/XML2JSON.xsl) does appear to generate that. Maybe we can look for an alternative stylesheet?

yep. It's a standard stylesheet which a had copied from somewhere (shame on me that I forgot to mention the source!) and which worked for a while. I'm just taking another one from the web - this time the source is inluded

**Comment by wowasa** [comment]
just for info: 
the web application is generating tsv and json for profile and collection table on demand from the XML report by XSLT. The stylesheets are AllProfileReport2TSV.xsl, AllCollectionReportToTSV.xsl and a generic stylesheet XML2JSON.xsl. 

We had the following errors:
1. incorrect root path and attribute name in  AllProfileReport2TSV.xsl
2. incorrect root path and an error in sub-template in AllCollectionReport.xsl
3. for XML2JSON.xsl I assume that either the stylesheet is not appropriate  for the new report structure anymore or the transformer has changed with the use of spring boot (before it was Saxon for sure). Further analysis is not necessary since we can find appropriate alternatives on the web

**Comment by wowasa** [comment]
fixed with version 6.0.2
